spalp.module

```diff
@@ -105,7 +105,9 @@ function spalp_get_app_ids(FieldStorageConfig $definition, ContentEntityInterfac
  * Implements hook_modules_installed().
  */
 function spalp_modules_installed($modules) {
-  $module_list = system_rebuild_module_data();
+  /** @var \Drupal\Core\Extension\ModuleExtensionList $module_list_service */
+  $module_list_service = \Drupal::service("extension.list.module");
```

Comment:
should be consistent with using single quotes unless double quotes are needed

Reply:
Done :)